### DISCUSSION TITLE
Public Key not available error During installation

### DISCUSSION THREAD

**Discussion by Mwenya930** [Ulauncher/Ulauncher#1201] [Q&A]
I've been trying to install Ulauncher for a couple of hours now, but I keep getting this error:

`The following signatures couldn't be verified because the public key is not available: NO_PUBKEY FAF1020699503176`

![screen](https://user-images.githubusercontent.com/125350821/218706630-bdca03ba-898e-4654-8894-d019aa16f66f.png)

Please how can I solve this?
Thank you.

**Comment by friday** [comment]
You use Debian, so follow the Debian instructions (not Ubuntu, but the expandable section in the Ubuntu section): https://ulauncher.io/#Download
